aviary/utils/process_input_decks.py

```diff
@@ -135,9 +142,15 @@ def parse_inputs(vehicle_deck, aircraft_values: AviaryValues(), meta_data=_MetaD
 
             elif var_name in guess_names:
                 # all initial guesses take only a single value
+                # get values from supplied dictionary
                 initial_guesses[var_name] = float(var_values[0])
                 continue
 
+            elif var_name.startswith('initial_guesses:'):
+                # get values labelled as initial_guesses in .csv input file
+                initial_guesses[var_name[len('initial_guesses:'):]] = float(var_values[0])
```

Comment:
I think you are trying to get the part of the string after `initial_guesses:`, right?
If so, using a `var_name.split(':')[-1]` is probably more clear

Reply:
Yes - that was the idea. 
I'm not really sure if this is necessary at all. 
The .csv input files currently has the initial_guesses listed in this way:
![image](https://github.com/OpenMDAO/Aviary/assets/165826178/7ad7e5a1-e44c-433f-b7ac-f81d5fc683aa)

I was wondering if it would be helpful for someone running 'parse_inputs' manually to be able to specify the initial_guesses in the input file without also needing to send the correct dictionary to the parse_inputs function.

I can either remove this section completely or change it as you suggest above.

Reply:
I think this is a good addition, but should be changed for clarity

Reply:
@crecine - I've made and committed your suggested change, tested it, and pushed to my fork... it seems that there's now a merge conflict - possibly just something to do with formatting... please let me know what I should do to correct it!

Reply:
resolved your merge conflict for you